src/utils/upload/index.ts

```diff
@@ -30,8 +31,10 @@ export function areFilesSame(
 ): boolean {
     if (
         existingFile.fileType === newFile.fileType &&
-        existingFile.creationTime === newFile.creationTime &&
-        existingFile.modificationTime === newFile.modificationTime &&
+        Math.abs(existingFile.creationTime - newFile.creationTime) <
+            A_SEC_IN_MICROSECONDS &&
```

Comment:
Shall we document why we changing this logic and comparing them like double with a precision of `A_SEC_IN_MICROSECONDS` (why not 10 or 100 MILLISECONDS?).

Reply:
Also, do we nee to change it for all files or just electronFiles?

Reply:
The value was chosen such that it could be much higher than what any user would keep to resist fingerprinting ([context](https://developer.mozilla.org/en-US/docs/Web/API/File/lastModified#reduced_time_precision)). I thought up to a second would be fine, but need to think what could be the smallest such value.

And yes we need it for all files since eventually the user will have a mix of uploads done by electronFiles and normal files and `existingFile` can be any of them.

Reply:
Got it. One second seems okay. We should some of this context as documentation, so that it will help us in future while maintaining this logic.